saleor/graphql/discount/mutations.py

```diff
@@ -66,7 +66,11 @@ def remove_catalogues_from_node(cls, node, input):
 
 class VoucherInput(graphene.InputObjectType):
     type = VoucherTypeEnum(
-        description="Voucher type: product, category shipping or value."
+        description=(
+            "Voucher type: product, category shipping or entire order. "
+            "Deprecated: PRODUCT, COLLECTION, CATEGORY are deprecated, "
```

Comment:
Here you have `deprecated` x2
Maybe something like: `"Deprecated fields: PRODUCT, COLLECTION, CATEGORY, use SPECIFIC_PRODUCT instead.`